src/combined-selectors/types.ts

```diff
@@ -1,5 +1,29 @@
+export type MatchType = 'name' | 'keyword' | 'description' | 'child';
+export type MatchOption = NameMatch | KeywordMatch | DescriptionMatch | ChildMatch;
+
+interface ChildMatch {
```

Comment:
When the search term matches a feature in the lowest level of the hierarchy, we add its parents to the search results. BUT, we don't want to indicate that the parents themselves match the description. Instead, we include them in the search results to display the hierarchical relationship in the tree.